features/step_definitions/register_steps.rb

```diff
@@ -0,0 +1,17 @@
+Given("I am on the index page") do
+  visit root_path
+end
+
+When("I fill in {string} with {string}") do |field, value|
+    fill_in field, with: value
+end
+
+Then("I should see the message {string}") do |content|
+    expect(page).to have_content content
+end
+
+Given("the following user exist") do |table|
+    table.hashes.each do |user|
+        FactoryBot.create(:user, user)
+   end
+end
```

Comment:
Code formatting is off in this file. You should use the same standard as the rest of your team. Check with them if they use 2 spaces or 1 tab, and use the same.